webapp/src/components/Modals/RentalListingModal/CreateOrEditListingStep/CreateOrEditListingStep.tsx

```diff
@@ -28,6 +30,9 @@ import {
 import { ManaField } from '../../../ManaField'
 import { Props } from './CreateOrEditListingStep.types'
 import styles from './CreateOrEditListingStep.module.css'
+import { config } from '../../../../config'
```

Comment:
Would you mind moving this up?
```suggestion
import { config } from '../../../../config'
```